src/Orleans/AssemblyLoader/AssemblyProcessor.cs

```diff
@@ -97,11 +97,14 @@ private static void ProcessAssembly(Assembly assembly)
             {
                 Logger.Verbose3("Processing assembly {0}", assemblyName);
             }
+
+#if !NETSTANDARD
```

Comment:
There is no reflection only load, so we won't have this scenario in .net standard (that's why it's not a TODO item at least here).
I'll review tomorrow if there's another reflection only load check that I incorrectly marked as TODO and fix it, thanks. The reality is that previously they were all just NETSTANDARD, and I recently started rebasing and crafting the commits so they add the NETSTANDARD_TODO one instead, so it is very likely that I missed a few where it made sense.